projects/4337-wallet.md

```diff
@@ -0,0 +1,29 @@
+# 4337 Wallet
+
+We are moving closer towards the finalisation of [EIP-4337](https://eips.ethereum.org/EIPS/eip-4337). A lot of effort by the core team has been put into designing the contracts, mempool, DOS protections etc. But the team haven't had time to properly support UserOps through one of the browser extension wallets.
+
+In this cohort, I intend to work on making a user-facing wallet which can then pave the way forward for the wallets in the Ethereum ecosystem. The larger goal is to test the eip enough and make it ready for everyday use. Once we get AAs into everyday use we can then easily work towards the depreciation of EOAs from the ecosystem.
+
+## Reading Material
+
+https://eips.ethereum.org/EIPS/eip-4337
+
+https://eips.ethereum.org/EIPS/eip-3074
+
+https://hackmd.io/@matt/r1neQ_B38
+
+https://github.com/eth-infinitism/account-abstraction
+
+https://github.com/eth-infinitism/bundler
+
+## My notes
+
+[[17.10.22] Onboarding to AA Wallet](https://github.com/eth-protocol-fellows/cohort-three/blob/master/notes/garvit.md#171022-onboarding-to-aa-wallet)
+
+[[11.10.22] Create cusotm SCW](https://github.com/eth-protocol-fellows/cohort-three/blob/master/notes/garvit.md#111022-create-cusotm-scw)
+
+[[03.10.22] Role of stake in paymasters](https://github.com/eth-protocol-fellows/cohort-three/blob/master/notes/garvit.md#031022-role-of-stake-in-paymasters)
+
+## My updates
+
+`TODO://`
```

Comment:
Is this intended to be filled in this PR or future PRs?
Asking just to be sure isn't a leftover!

Reply:
As and when I will put updates in the `development-updates.md` I will add the reference here too. So that this project file can be used as a single source of information.

Reply:
So yes future PRs.